source/NVDAObjects/UIA/__init__.py

```diff
@@ -369,6 +370,14 @@ def findOverlayClasses(self,clsList):
 		# #5942: In recent Windows 10 Redstone builds (14332 and later), Microsoft rewrote various dialog code including that of User Account Control.
 		if self.UIAIsWindowElement and UIAClassName in ("#32770","NUIDialog", "Credential Dialog Xaml Host"):
 			clsList.append(Dialog)
+		# 6241: Try detecting all possible suggestions containers and search fields scattered throughout Windows 10.
+		if self.UIAElement.cachedAutomationID in ("TextBox", "SearchTextBox"):
+			clsList.append(SearchField)
+		try:
+			if self.parent.UIAElement.cachedAutomationId.lower()=="suggestionslist":

```

Comment:
This could be very costly and could end up being recursive -- I had made the same mistake with Microsoft Edge code earlier this year. Items within a suggestionList don't have a useful UIA className or automationID?